base64ct/src/alphabet.rs

```diff
@@ -70,7 +71,7 @@ pub trait Alphabet: 'static + Copy + Debug + Eq + Send + Sized + Sync {
         ret
     }
 
-    /// Encode 3-bytes of a Base64 message.
+    // Encode 3-bytes of a Base64 message.
```

Comment:
This seems unnecessary

Reply:
Oh, I missed that.  Sorry.  Fixed in latest commit.